### ISSUE TITLE
[Fortress] no roll and no medallions in Reikwald

### ISSUE THREAD

**Issue by elk76** [WarEmu/WarBugs#13605]
Greetings,
last night in Reickwald, we (destro) attacked the fort. We killed the lord and many members of the warband did not reviced medallions for the lock and did not roll for bags
cheers

**Comment by EssentialOrder** [comment]
Same here 3rd fortress in row, Butcher pass today  no roll no medalions i still have 52 medalions . You can check logs i miss 12 medalions 
Engineer  Frostrock

**Comment by sioding** [comment]
This should be fixed with todays hotfix. I leave this one open - please put in any helpful information after the next open fort when you do not get your reward.

**Comment by ChicagoJoe999** [comment]
Is it related to caps for rewards? With x2 maybe over caps.

**Comment by sioding** [comment]
This should be fixed

**Comment by mlubrick** [comment]
Might need to re-open this issue.

Happened to me again today capturing Shining Way ~3:15 p.m. MST.  My log shows dmg on the Lady and her dmg on me, she dropped (with normal lag) and then the roll visual came up.  However no renown tick and no meds awarded.  I know others within the same Fortress received both, so not sure why mine did not trigger.  Figured I'd submit in case others have same issue.

Note - I was getting renown ticks for player kills.
Note2 - Not sure if it matters, but I joined Fort during stage 3, we had Lady down to ~10% when server crashed first time (assumed b/c of Reikwatch Fort capture at same time).  Once able to log back in, had to wait until reservations were done but I did get back in 2nd time.  Took down Lady, but no tick/meds.

Thanks!
/ign Boruch
